image/templates/helm/stackrox-central/templates/02-config-controller-01-rbac.yaml

```diff
@@ -0,0 +1,44 @@
+{{- include "srox.init" . -}}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: config-controller-manager-role
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "role" "config-controller-manager-role") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "role" "config-controller-manager-role") | nindent 4 }}
+rules:
+- apiGroups:
+  - config.stackrox.io
+  resources:
+  - policies
+  - policies/status
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
This should definitely be a `RoleBinding` since we want the config-controller to only be able to see policy CRs in its own namespace.